src/runtime/bootstrap-lazy.ts

```diff
@@ -185,16 +185,18 @@ export const bootstrapLazy = (lazyBundles: d.LazyBundlesRuntimeData, options: d.
     });
   });
 
+  dataStyles.innerHTML = SLOT_FB_CSS;
+  dataStyles.setAttribute('data-styles', '');
+  head.insertBefore(dataStyles, metaCharset ? metaCharset.nextSibling : head.firstChild);
```

Comment:
For my own understanding, why do we hoist this out of [this conditional](https://github.com/ionic-team/stencil/pull/5028/files#diff-391b4dd275b4968d6629df297b178c24ebb07597f843b3f1ce5a3d6f4de82e16R192)?

Reply:
Because the styles for `slot-fb` should not be contingently created and applied based on the `invisiblePrehydration` config option. So, we always create and insert the style element, but will continue to only inject the `invisiblePrehydration` styles based on that config value.

Also added a quick comment in the code to help clarify that a bit.

Reply:
Can you expound on that for me? `invisiblePrehydration` is `true` by default, which meant we were setting the `innerHTML` and `data-styles` by default for most folks only if `hydratedClass` or `hydratedAttribute` were set (or both). In fact, before `invisibleHydration` was added (a3d2928d), this section looked like the following in ac49d7c3:
```ts
  if (BUILD.hydratedClass || BUILD.hydratedAttribute) {
    visibilityStyle.innerHTML = cmpTags + HYDRATED_CSS;
    visibilityStyle.setAttribute('data-styles', '');
    head.insertBefore(visibilityStyle, metaCharset ? metaCharset.nextSibling : head.firstChild);
  }
```

Reply:
That sill happens. So now, if you have a component using a slot outside the Shadow DOM (i.e. `scoped` or no encapsulation) and have the conditions for `invisiblePrehydration`, you get styles like the following:

```css
my-component {
  visibility:hidden
}
.hydrated {
  visibility:inherit
}
slot-fb {
  display: contents
}
slot-fb[hidden] {
  display: none
}
```